### PR TITLE
Docs web complete

### PR DESCRIPTION
**Pull request by MiheerTamkhane**

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+@import url("/components/root.css");
+
+/* ct-round class makes img round  */
+
+.ct-avatar {
+  width: 32px;
```
Comment (nshanbhag20001412) [inline components/avatar/avatar.css:6]: start using rem wherever possible

File: components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  color: #fff;
+}
+.ct-btn.ct-green {
+  background: var(--green-500);
+  color: var(--body);
+  transition: all 250ms ease;
+}
+.ct-btn.ct-green:hover {
+  background: var(--green-700);
+  color: #fff;
```
Comment (nshanbhag20001412) [inline components/button/button.css:38]: color variable may be?

**Review by nshanbhag20001412 — COMMENTED** [review]
Reviewing 38 files becomes tedious. If possible try to break it down

**Comment by MiheerTamkhane** [comment]
> eviewing
okay i will, actually i have 2 repos comps and comps docs
I pushed all the comps in docs once at time thats why.
I'll do it 
Thank you.

File: components/avatar/avatar.css (hunk 1 above)
Comment (MiheerTamkhane) [reply components/avatar/avatar.css:6]: Oh thanks for that neha

File: components/button/button.css (hunk 2 above)
Comment (MiheerTamkhane) [reply components/button/button.css:38]: Actully I explicitly want that white . color but I'll look into it or create new var for white I think. Thanks for that too

**Pull request merged**